src/frontend/src/icp/workers/icrc-wallet.worker.ts

```diff
@@ -61,16 +61,20 @@ const mapTransaction = ({
 	return mapIcrcTransaction({ transaction, identity });
 };
 
-const scheduler: IcWalletScheduler<
+// Exposed for test purposes
```

Comment:
i don't think it is necessary this comment, right? i mean, i exposed a lot of unnecessary ones just for testing, eheh

Or shall we add this comment in the others too?

Reply:
No strong opinion, I dislike exposing anything just for testing purpose because the production bundles ends with module features exposed for no reason, that's why I compromise with adding a comment which basically mean for me: "I had no other choice".